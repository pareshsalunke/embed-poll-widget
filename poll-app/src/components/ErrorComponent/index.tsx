import React from 'react';

const ErrorComponent = () => (
  <div>
    <h4>Error: Loading Survey</h4>
  </div>
);
export default ErrorComponent;