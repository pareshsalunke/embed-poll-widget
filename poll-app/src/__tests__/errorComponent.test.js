import React from 'react';
import ErrorComponent from '../components/ErrorComponent';
import { render } from '@testing-library/react';

describe('Error component', () => {
  let Comp;
  beforeEach(() => {
    Comp = () => {

      return (
        <ErrorComponent />
      );
    }
  });

  test('error component mounts', () => {
    expect(Comp()).toMatchSnapshot();
  });

  test('error component text', () => {
    const { getByText } = render(<Comp />);
    expect(getByText("Error: Loading Survey")).toBeInTheDocument();
  });
})