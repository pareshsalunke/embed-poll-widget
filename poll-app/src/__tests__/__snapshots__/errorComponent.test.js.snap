// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error component error component mounts 1`] = `<ErrorComponent />`;
